rosidl_runtime_cpp/test/test_bounded_vector.cpp

```diff
@@ -50,3 +52,39 @@ TEST(rosidl_generator_cpp, bounded_vector_rvalue) {
   ASSERT_EQ(vv[0], 1);
   ASSERT_EQ(vv[1], 2);
 }
+
+TEST(rosidl_generator_cpp, bounded_vector_comparisons) {
+  rosidl_runtime_cpp::BoundedVector<int, 2> v;
+  ASSERT_EQ(v, v);
+  const decltype(v) vv;
```

Comment:
@jwakely why not 

```c++
  rosidl_runtime_cpp::BoundedVector<int, 2> v, vv;
```

above? It's a tiny bit clearer.

Reply:
I was just copying what was already on line 49.

Reply:
If you don't like that style, you should probably change the existing tests so people don't try to be consistent with them.

Reply:
Hmm, I see. Not happy with either, but fine to leave as-is.